reconstruct/reconstruct.go

```diff
@@ -69,6 +72,25 @@ func LikelihoodsToBase(likelihoods []float64, nonBiasBaseThreshold float64, bias
 	return answer
 }
 
+// LikelihoodsToPbase converts and normalises a slice of probabilities for one position of a sequence. That position refers to a specific base.
+func LikelihoodsToPBase(likelihoods []float64, nonBiasBaseThreshold float64, biasBase dna.Base, highestProbThreshold float64) pDna.Float32Base {
```

Comment:
Are the `nonBiasBaseThreshold float64, biasBase dna.Base, highestProbThreshold float64` inputs needed, or can they be deleted since pDna output might not have a bias?